### DISCUSSION TITLE
NewFailoverClusterClient and DB selection

### DISCUSSION THREAD

**Discussion by mdouchement** [redis/go-redis#1733] [General]
Hello, I'm a little bit lost with the experimental `NewFailoverClusterClient` client.
It's made to connect to a Redis Sentinel but does not allow to select database like a Redis Cluster.

Since `NewFailoverClusterClient` returns a `ClusterClient` this behavior seems legit but I think we should be able to use the databases feature.

**Comment by Queetinliu** [comment]
you can put DB in redis failoveroptions.
